docs/using.md

```diff
@@ -1,166 +1,484 @@
+
+
+<a id="orgdcd1898"></a>
+
+## What is load generation?
+
+Genny is a **load generation** tool. It is used as a client in [load tests](https://en.wikipedia.org/wiki/Load_testing) to perform a large amount of work against a system under test, measuring the client-side visible responses of said test subject. These measurements may represent things like latency, number of operations performed, how many errors occurred, etc. The procedure Genny uses to perform the work should be as deterministic and repeatable as possible. This procedure is called a **workload**.
+
+Results of a load test can inform developers as to the performance of the test subject in stressful situations. How does a system handle increasing numbers of connections performing conflicting operations? How does a system handle a large number of users performing one operation, then simultaneously switching to another in an instant? How does one user initiating a long-running, expensive operation affect the latencies observed by other users? How does the system respond when put under load much higher than originally intended (stress testing)? All these situations can be simulated with Genny.
+
+<a id="orgc7988ae"></a>
+
+### What other load generation tools are there?
+At MongoDB, we've historically used [benchrun-based](https://github.com/10gen/workloads) workloads. This legacy tooling was deprecated in favor of Genny in order to provide:
+
+- A standardized [interface](#org3610c67) for workload authoring in yaml.
+- A [phase mechanism](#orgb655d69) for organizing workload execution.
+- More reusability via yaml-configurable [actors](#org51d4d33) with a consistent C++ [authoring mechanism](#org7e6c6bd).
+- High-fidelity intrarun [time series outputs](#orgec88ad4).
+- Lack of dependency on the legacy mongo shell.
+
+For the above reasons, we encourage users to write new workloads in Genny.
+
+<a id="orgc7904ae"></a>
+
+## What is the system under test?
+
+Genny usually expects to be given a connection string to a system under test. While there's nothing preventing a user from writing a "workload" that spawns the system under test, there is not first-class support for such behavior. Genny is explicitly <span class="underline">not</span> a test orchestrator. For test orchestration needs, consider using [DSI](https://github.com/10gen/dsi/).
+
+By default, Genny will try to connect to a MongoDB server at `localhost:27017`. To pass a MongoDB connection string to Genny, use the `-u` option:
+
+    ./run-genny workload -u [arbitrary_URI] src/workloads/docs/InsertWithNop.yml
+
+For more details on how Genny handles connections, see [Connecting to the Server](#orgd6b0450).
+
+
+<a id="org3610c67"></a>
+
+## What is a workload?
+
+A **workload** is a repeatable procedure that Genny uses to generate load against a system under test. Genny workloads are written in yaml configs that describe how **actors** move through **phases**. This section describes each of these.
```

Comment:
(super nit: but I always capitalize Actors to reinforce that it's a genny-defined construct. They're not traditional message-passing actors from e.g. elixir. I'm totally fine to drop that as an anachronism if it comes across as too pedantic though.)